integration/mediation-tests/tests-patches/src/test/resources/testng.xml

```diff
@@ -255,6 +255,7 @@
     <test name="Payload Factory" preserve-order="true" verbose="2" parallel="false">
         <classes>
             <class name="org.wso2.carbon.esb.mediators.payloadFactory.ESBJAVA3573PayloadFormatWithBeginHtmlTagArgument"/>
+            <!--<class name="org.wso2.carbon.esb.mediators.payloadFactory.ESBJAVA5030PayloadFormatArgumentWithTrailingSpaceTestCase"/>--> <!-- TODO uncomment once dependencies changes to SNAPSHOT-->
```

Comment:
EI is not getting build on top of synapse snapshot versions, hence public branch fix doesn't come to the EI snapshot build, hence it's commented, later when it is migrated to snapshot dependency we can uncomment this, (TODO is for that)